MANIFEST.in

```diff
@@ -1,29 +1,17 @@
 include README.rst
 include CHANGES.rst
 
-include ez_setup.py
-include ah_bootstrap.py
-include setup_helpers.py
-include setup.cfg
-
-recursive-include *.pyx *.c *.pxd
-recursive-include docs *
 recursive-include licenses *
-recursive-include cextern *
-recursive-include scripts *
 recursive-include asdf-standard/schemas *.yaml
 recursive-include asdf-standard/reference_files *
 
-# Python version specific compatibility packages, normally excluded by default
-# depending on the Python version doing the building
-recursive-include asdf/compat *.py
```

Comment:
I checked our supported Python versions and these files make it into the sdist without this explicit include